recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"use_boost": [True, False], "use_openssl": [True, False]}
+    default_options = {"use_boost": False, "use_openssl": False}
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
+
+    def requirements(self):
```

Comment:
This is missing form the stiffstream recipe
```suggestion
    def requirements(self):
        self.requires("zlib/1.12.1")
```

Reply:
Looks like it should be optional. I will add an option.

Reply:
There's also prce2 support that's an option

Reply:
restinio support pcre1 and pcre2, how does the option should looks like ? Something like `with_pcre: [1, 2, None]` ?

Reply:
That sounds reasonable